### DISCUSSION TITLE
win7老电脑哦，我是少下了什么东西吗

### DISCUSSION THREAD

**Discussion by SillyAndyZeng** [mikumifa/biliTickerBuy#427] [提问专区]
![捕获](https://github.com/user-attachments/assets/a9ad512e-107b-44d1-8ce6-0e4afcf5f311)
已经重装过一遍了，甚至没有发现.dll文件在哪
救我，我不是很懂电脑……

**Comment by mikumifa** [comment]
不兼容win10以下。。。
